source/includes/databases-collections/databases-collections.php

```diff
@@ -0,0 +1,93 @@
+<?php
+require 'vendor/autoload.php';
+
+use MongoDB\BSON\Document;
```

Comment:
```suggestion
use MongoDB\BSON\Document;
```

This class isn't referenced in the code.